### PR TITLE
Fix missing tooltip

### PR DESCRIPTION
**Pull request by rupato-deriv**

### CODE REVIEW

File: src/botPage/view/deriv/components/tooltip/tooltip.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/* eslint-disable indent */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Tooltip = props => {
```
Comment (salarhfz-fs) [inline src/botPage/view/deriv/components/tooltip/tooltip.js:5]:
```suggestion
const Tooltip = ({ delay, children, direction, content }) => {
```

File: src/botPage/view/deriv/components/tooltip/tooltip.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    return (
+        <div className="bot-tooltip" onMouseEnter={showTip} onMouseLeave={hideTip}>
+            {props.children}
+            {active && <div className={`bot-tooltip__popover ${props.direction || 'top'}`}>{props.content}</div>}
+        </div>
+    );
+};
+
+Tooltip.propTypes = {
```
Comment (salarhfz-fs) [inline src/botPage/view/deriv/components/tooltip/tooltip.js:28]: We may add `children` here as well.

File: static/css/_constants.scss [hunk 3]
```diff
@@ -24,4 +24,7 @@ $COLOR_RED           : #c03;
 $COLOR_YELLOW        : #fef1cf;
 $COLOR_GRAY          : #dedede;
 $COLOR_DARK_GRAY     : #44484c;
-/* stylelint-enable color-no-hex */
\ No newline at end of file
+/* stylelint-enable color-no-hex */
+
+$TOOLTIP-MARGIN: 10px;
+$TOOLTIP-ARROW-SIZE: 6px;
```
Comment (salarhfz-fs) [inline static/css/_constants.scss:30]: EoF

File: static/css/deriv/popover.scss [hunk 4]
```diff
@@ -23,6 +23,7 @@
             display: block;
             width: 0;
             z-index: 1;
+            display: none;
```
Comment (sara-fs) [inline static/css/deriv/popover.scss:26]: when we are not applying any condition, when the span is shown?

File: src/botPage/view/deriv/components/popover/popover.jsx [hunk 5]
```diff
@@ -5,7 +5,7 @@ const Popover = ({ children , content })=>{
     const [is_hovered, updateIsHovered] = React.useState(false);
     
     return(
-        <div
+        <span
```
Comment (salarhfz-fs) [inline src/botPage/view/deriv/components/popover/popover.jsx:8]: Why is this changed to a `span`?

File: src/botPage/view/deriv/layout/ToolBox/ToolBox.jsx [hunk 6]
```diff
@@ -86,79 +92,110 @@ const ToolBox = ({ blockly }) => {
...
+      </Popover> */}
+      {/* <Popover content={translate("Run the bot")} position="bottom">
+        <button id="runButton" className="toolbox-button icon-run" />
+      </Popover>
+      <Popover content={translate("Stop the bot")} position="bottom">
+        <button id="stopButton" className="toolbox-button icon-stop" />
+      </Popover>
+      <Popover content={translate("Show log")} position="bottom">
+        <button id="logButton" className="toolbox-button icon-info" />
+      </Popover> */}
```
Comment (salarhfz-fs) [inline src/botPage/view/deriv/layout/ToolBox/ToolBox.jsx:184]: Should re remove the comments?

File: src/botPage/view/deriv/layout/ToolBox/ToolBox.jsx [hunk 7]
```diff
@@ -175,3 +212,6 @@ ToolBox.propTypes = {
 };
 
 export default ToolBox;
+
+
+
```
Comment (salarhfz-fs) [inline src/botPage/view/deriv/layout/ToolBox/ToolBox.jsx:217]: 3 new lines at the end. 1 should be enough.

File: static/css/bot.scss [hunk 8]
```diff
@@ -204,3 +205,6 @@ label {
 .notifyjs-corner {
     z-index: 10001 !important;
 }
+
+// 
+
```
Comment (salarhfz-fs) [inline static/css/bot.scss:210]: What is this for?

File: src/botPage/view/deriv/components/popover/popover.jsx (hunk 5 above)
Comment (rupato-deriv) [reply src/botPage/view/deriv/components/popover/popover.jsx:8]: because div takes display block by default and span takes inline block by default

**Review by salarhfz-fs — APPROVED** [review]

**Review by mehrzad-fs — APPROVED** [review]
